src/color/p5.Color.js

```diff
@@ -271,11 +269,9 @@ p5.Color.prototype.toString = function(format) {
  *   backgroundColor.setRed(128 + 128 * sin(millis() / 1000));
  *   background(backgroundColor);
  * }
+ * describe('canvas with gradually changing background color');
```

Comment:
Here's where the error is! describe() must be in the setup or draw!

Reply:
Oh!  Whoopsie-daisy!

(Thanks!)  I’ll fix it shortly...

Reply:
K, I think I’ve got it.